Source/WebCore/platform/graphics/ca/PlatformCALayer.mm

```diff
@@ -210,8 +210,8 @@
 
 LayerPool* PlatformCALayer::layerPool()
 {
-    static LayerPool* sharedPool = new LayerPool;
-    return sharedPool;
+    static NeverDestroyed<UniqueRef<LayerPool>> sharedPool = makeUniqueRef<LayerPool>();
+    return sharedPool->ptr();
```

Comment:
This means it's always non-null, right? I think that means some of the other code can be cleaned up as well as we currently null-check the return value.

Reply:
Unfortunately we can't do that because `Source/WebKit/WebProcess/WebPage/RemoteLayerTree/PlatformCALayerRemote.mm` which subclasses this class can return `nullptr` here.